src/test/java/org/broadinstitute/hellbender/utils/IndexUtilsUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.tribble.index.Index;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class IndexUtilsUnitTest extends BaseTest{
+
+    @DataProvider(name= "okFeatureFiles")
+    public Object[][] okFeatureFiles() {
+        return new Object[][] {
+                { new File(getToolTestDataDir(), "test_variants_for_index.vcf")},
+                { new File(getToolTestDataDir(), "test_variants_for_index.gvcf.vcf")},
+                { new File(getToolTestDataDir(), "test_bed_for_index.bed")},
```

Comment:
added that and tests for tribble and tabix indices (loading them and failing to load them when appropriate)